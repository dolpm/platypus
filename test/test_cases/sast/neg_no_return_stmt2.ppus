pipe main [] |> unit {
  while (true) {
    if (true) {
      |> ();
    }
  } 
}