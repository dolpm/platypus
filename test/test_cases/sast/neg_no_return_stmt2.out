Fatal error: exception Failure("return value not provided in main")
